packages/flutter_tools/lib/src/devfs.dart

```diff
@@ -722,6 +723,14 @@ class DevFS {
       }
     }
     if (sb != null) {
+      if (previewDart2) {
```

Comment:
Why not
if (!previewDart2 && (sb != null)) {
  .....
}

Reply:
Refactored slightly.